website/content/en/preview/tasks/amitasks.md

```diff
@@ -0,0 +1,157 @@
+  - id: ami-052c9ea013e6e3567
+```
+
+See the [**spec.amiSelectorTerms**]({{< relref "../concepts/nodeclasses/#specamiselectorterms" >}}) section of the NodeClasses page for details. 
+Keep in mind, that this could prevent you from getting critical security patches when new AMIs are available, but it does give you control over exactly which AMI is running.
+
+### Task 2: Prevent Karpenter from disrupting nodes
+
+There are several ways you can prevent Karpenter from disrupting nodes that it manages, to mitigate the risk of an untested AMI from being deployed.
+
+* **Set Pods to not allow disruption**: When you run pods from a Deployment spec, you can set `karpenter.sh/do-not-disrupt` to true on that Deployment.
+This will prevent the node that pod is running on from being disrupted while the pod is running (see [**Pod-level Controls**]({{< relref "../concepts/disruption/#pod-level-controls" >}}) for details).
+This can be useful for things like batch jobs, which you want to run to completion and never be moved.
+For example:
+
+    
+```bash
+    apiVersion: apps/v1
+    kind: Deployment
+    spec:
+      template:
+        metadata:
+          annotations:
+            karpenter.sh/do-not-disrupt: "true"
+```
+
+* **Set nodes to not allow disruption** In the NodePool spec, you can set `karpenter.sh/do-not-disrupt` to true.
+This prevents any nodes created from the NodePool from being considered for disruption (see [**Example: Disable Disruption on a NodePool**]({{< relref "../concepts/disruption/#example-disable-disruption-on-a-nodepool" >}}) for details).
+For example:
+
+```bash
+    apiVersion: karpenter.sh/v1beta1
+    kind: NodePool 
+    metadata:
+      name: default
+    spec:
+      template:
+        metadata:
+          annotations: # will be applied to all nodes
+            karpenter.sh/do-not-disrupt: "true"
+```
+
+Keep in mind that these are not permanent solutions and cannot prevent all node disruptions, such as disruptions resulting from failed node health checks or the instance running the node going down.
+Using only the methods to prevent disruptions described here, you will not prevent new AMIs from being used if an unintended disruption of a node occurs, unless you have already locked down specific AMIs to use.
+
+### Task 3: Control the pace of node disruptions
+
+To help prevent the possibilities of a new AMI being deployed to all your nodes and breaking all of your workloads, you can enable Karpenter [**Disruption Budgets**]({{< relref "../concepts/disruption/#disruption-budgets " >}}).
+Disruption Budgets limit when and to what extent nodes can be disrupted.
+You can prevent disruption based on nodes (a percentage or number of nodes that can be disrupted at a time) and schedule (excluding certain times from disrupting nodes).
+You can set Disruption Budgets in a `NodePool` spec.
+Here is an example:
+
+```bash
+disruption:
+  consolidationPolicy: WhenUnderutilized
+  consolidateAfter: 30s
```

Comment:
`consolidateAfter` is currently only available when the `consolidationPolicy` is `WhenEmpty`.

Reply:
Good catch. I'll change the  it `consolidationPolicy` to `WhenEmpty`.